marketplace/deployer_helm_tiller_base/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM launcher.gcr.io/google/debian9
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    bash \
+    ca-certificates \
+    curl \
+    gettext \
+    git \
+    jq \
+    procps \
+    python \
+    python-pip \
+    python-setuptools \
+    python-yaml \
+    wget \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install wheel \
+    && pip install google-cloud-storage
+
+RUN wget -q -O /bin/kubectl \
+        https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/linux/amd64/kubectl \
+    && chmod 755 /bin/kubectl
+
+RUN mkdir -p /bin/helm-downloaded \
+    && wget -q -O /bin/helm-downloaded/helm.tar.gz \
+        https://storage.googleapis.com/kubernetes-helm/helm-v2.10.0-linux-amd64.tar.gz \
+    && tar -zxvf /bin/helm-downloaded/helm.tar.gz -C /bin/helm-downloaded \
+    && mv /bin/helm-downloaded/linux-amd64/helm /bin/ \
+    && rm -rf /bin/helm-downloaded \
+    && helm init --client-only \
+    && helm plugin install https://github.com/rimusz/helm-tiller \
+    && helm repo add cloud-marketplace-tools-charts https://cloud-marketplace-tools-charts.storage.googleapis.com
+
+COPY marketplace/deployer_util/* /bin/
+COPY marketplace/deployer_helm_tiller_base/bin/* /bin/
+
+RUN mkdir -p /data/chart
+
+ENTRYPOINT ["/bin/bash", "/bin/deploy.sh"]
+
+ONBUILD COPY . /tmp/onbuild
+
+ONBUILD ARG HELM_DEPENDENCY_BUILD=false
+ONBUILD ENV HELM_DEPENDENCY_BUILD $HELM_DEPENDENCY_BUILD
+
+# Partner directory structure assertions.
+ONBUILD RUN if [ "$(find /tmp/onbuild -maxdepth 1 -mindepth 1 -type d -printf '\n' | wc -l)" -ne 1 ]; then \
```

Comment:
can we make sure that the directory that this expects is the same as in the [guide](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/building-deployer-helm.md#download-the-helm-chart). We should keep the convention consistent so that we can just point partners to the doc.

The internal of the tiller deployer can be different from the non-tiller one (that's an advantage of the onbuild paradigm that we discussed).

Reply:
Done!